frontend/public/locales/en-US/common.json

```diff
@@ -65,6 +65,9 @@
     "analyticsDocumentation": "Analytics Documentation",
     "feedback": "Feedback",
     "signOut": "Sign out",
+    "updateAvailable1": "New version {{version}} is available. Click to upgrade.",
+    "updateAvailable2": "New version {{version}} is available. Contact your administrator to upgrade.",    
```

Comment:
A new version {{version}} is now available. Please contact your administrator to upgrade.